docs/products/data_product/how_to_guides/design.md

```diff
@@ -216,142 +216,51 @@ Let’s see how you can set up the Data Product.
 
 Begin by creating a manifest file that will hold the configuration details for your Data Product. The structure of the Data Product manifest file is provided below.
 
-=== "version: v1alpha"
```

Comment:
I don't see all the updated changes, you have to change the description of all the attributes on that page as well.